README.md

```diff
@@ -1326,6 +1327,10 @@ private func makeLocationManager() -> CLLocationManager {
 * Unused (dead) code, including Xcode template code and placeholder comments should be removed.
 * Don't use `#imageLiteral` or `#colorLiteral`, use Redbreast.
 
+## RxSwift
+
+Since RxSwift is quite complex on its own, we have created a secondary [Readme](RxSwift.md) file for it.
```

Comment:
I would rather go with something different than "secondary" - people may misunderstood and think where is the second readme about RxSwift.

So, maybe something like; "Due to the reactive nature of RxSwift library and different approaches, we defined our own code style for it."